libraries/types/vote/src/vote.cpp

```diff
@@ -82,15 +97,26 @@ void Vote::validate(size_t dpos_total_votes_count, size_t sortition_threshold) c
   }
 }
 
-bytes Vote::rlp(bool inc_sig) const {
+bytes Vote::rlp(bool inc_sig, bool inc_weight) const {
```

Comment:
do we really need inc_weight option? Maybe I am missing something, but it is set true everywhere

Reply:
I think we need it for next/soft votes, where we save them to DB and later use them, but I am planning to do big refactor of everything related to votes, so I think I will remove it in the future